### DISCUSSION TITLE
Select 组件开启Search时，使用popupScoll自定义加载下一页数据，使用按键如何控制不跳回第一行

### DISCUSSION THREAD

**Discussion by chenzequ** [ant-design/ant-design#36689] [Q&A]
我重新整理了一下问题，当我使用 Select 组件 进行懒加载下一页数据时， 也就是 state.data.push(...newData) 时，我使用了向下按键，监听了 popupScoll 滚动到底部， 此时 滚动窗口处于最底部，但是 option 焦点变成了第一个option .... 可能是由于option数据重新渲染，导致了 active的索引自动跳回到了第一位，正常情况下这样是没有问题的，也是方便的。不过我这种情况下就很奇怪了，看看是不是能够解决这个问题。

**Comment by valcosmos** [comment]
您好，可以看下这个例子。

[https://ant.design/components/select-cn/#components-select-demo-search-box](https://ant.design/components/select-cn/#components-select-demo-search-box)
